components/Common/ConfirmDelete/index.js

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+import StyledButton from "../StyledButton";
+import { ButtonContainer, BackDrop } from "../../../styles";
+
+export default function ConfirmDelete({
+  projectId,
+  cancelDelete,
+  onDelete,
+  deleteYarnStockStatus,
+  yarnId,
+  handleDeleteExistedYarn,
+  cancelDeleteYarnStock,
+}) {
+  let deleteItem = "";
+  if (deleteYarnStockStatus) {
+    deleteItem = "yarn";
+  } else {
+    deleteItem = "project";
+  }
+
```

Comment:
Just as hint how you could shorten this code:
`let deleteItem = "project"
if (deleteYarnStockStatus) deleteItem = "yarn"`

Reply:
Thanks David, later i will use this again for needle delete feature,  there wil be at last three options
It will be like this:
let deleteItem = ''
if(deleteYarnStockStatus) {deleteItem = 'yarn'} else if(deleteNeedleStockStatus){deleteItem = 'needle'} else {deleteItem = 'project'}